specification/billing/resource-manager/Microsoft.Billing/privatepreview/2020-11-01-privatepreview/examples/BillingActivityList.json

```diff
@@ -0,0 +1,43 @@
+{
+  "parameters": {
+    "api-version": "2020-11-01-privatepreview",
+    "billingAccountName": "{billingAccountName}",
+    "startDate": "{startDate}",
+    "endDate": "{endDate}",
+    "activityLogEventCategory": "{activityLogEventCategory}"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "id": "/providers/Microsoft.Billing/billingAccounts/7303440/billingActivityList",
```

Comment:
ID should include the name of the billing account resource. Alternatively, consider modeling BillingActivity as a non-ARM resource so it doesn't have id/name properties at all.

Reply:
7303440 here is the name of the billing Account in Id. We intend to keep BillingActivity response an Arm type only.